### PR TITLE
Handle eventhub cardinality

### PR DESCRIPTION
**Pull request by maiqbal11**
Fixes https://github.com/Azure/azure-functions-python-worker/issues/411

### CODE REVIEW

**Comment by pragnagopa** [comment]
Can you please send a separate PR for updating proto file?

**Comment by maiqbal11** [comment]
@pragnagopa, rebased on top of the new `dev` after updating protobuf.

File: tests/endtoend/eventhub_batch_functions/eventhub_multiple/function.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "scriptFile": "__init__.py",
+
+  "bindings": [
+    {
+      "type": "eventHubTrigger",
+      "name": "events",
+      "direction": "in",
+      "cardinality": "many",
+      "dataType": "string",
```
Comment (pragnagopa) [inline tests/endtoend/eventhub_batch_functions/eventhub_multiple/function.json]: do not specify dataType. String is default

**Comment by maiqbal11** [comment]
> no changes needed in the worker or library to convert TypedDataCollection to a list of events? Can you please point me code where this conversion happens?

Here is the corresponding library code: https://github.com/Azure/azure-functions-python-library/pull/24

File: tests/endtoend/eventhub_batch_functions/eventhub_multiple/function.json (hunk 1 above)
Comment (maiqbal11) [reply tests/endtoend/eventhub_batch_functions/eventhub_multiple/function.json]: Looks like we are failing on a JSON deserialization step when `dataType` is not specified as `string`. The SDK must be assuming that we are writing a JSON deserializable string by default. I was able to repro in Node. Will open an issue against the SDK.

Comment (maiqbal11) [reply tests/endtoend/eventhub_batch_functions/eventhub_multiple/function.json]: Opened an issue on the EventHub extension: https://github.com/Azure/azure-functions-eventhubs-extension/issues/17.

**Pull request merged**